monkestation/code/modules/blueshift/items/ash_walker.dm

```diff
@@ -61,6 +61,10 @@
 	icon_state = "mesh"
 	var/list/static/seeds_blacklist = list(
 		/obj/item/seeds/lavaland,
+		/obj/item/seeds/gatfruit,
+		/obj/item/seeds/tree/money,
+		/obj/item/seeds/random,
+		/obj/item/seeds/seedling/evil,
```

Comment:
oh can you also add `/obj/item/seeds/spliced` to this? it's completely useless when spawned outside the context of a splicer, so it just clogs up the pool of valid seeds